src/main/cli/cli.c

```diff
@@ -1409,14 +1410,34 @@ static int cliParseSerialMode(const char *tok)
     return mode;
 }
 
+static int cliParseSerialOptions(const char *tok)
+{
+    int options = SERIAL_NOT_INVERTED;
+
+    if (strcasestr(tok, "8N1")) {
+        options |= SERIAL_STOPBITS_1 | SERIAL_PARITY_NO;
+    }
+    if (strcasestr(tok, "8N2")) {
+        options |= SERIAL_STOPBITS_2 | SERIAL_PARITY_NO;
+    }
+    if (strcasestr(tok, "8E1")) {
+        options |= SERIAL_STOPBITS_1 | SERIAL_PARITY_EVEN;
+    }
+    if (strcasestr(tok, "8E2")) {
```

Comment:
For completeness' sake we could add here the 2 'odd' options as well.

Reply:
The reasoning around only including these options was because these are the only ones currently available when calling openSerialPort:

https://github.com/betaflight/betaflight/blob/271c8799ddf9cd8cd070420d3a85d54585a3cdce/src/main/drivers/serial.h#L40

I could look at adding ODD parity as an option, but that would likely be a much larger change?

Reply:
Lol, I wasn't aware that we don't support ODD. All good then.